Userland/Libraries/LibVideo/VP9/TreeParser.h

```diff
@@ -29,8 +30,17 @@ class TreeParser {
             int m_value;
         };
 
-        TreeSelection(int const* values);
-        TreeSelection(int value);
+        constexpr TreeSelection(int const* values)
```

Comment:
I might be missing something but the fact that this class doesn't know how many elements are in this array weirds me out.

yakplus on this one I guess.

Reply:
Yeah, might be worth coming back to this, that's how it was before. Not sure if the class was even necessary here since trees with single values can just be decoded with a different function call, that's what other decoders do. Another thing to refactor, I suppose.